# header
std_msgs/Header header

src2_object_detection/Box[] boxes
